### DISCUSSION TITLE
Version formatting mismatch between release assets vs container images

### DISCUSSION THREAD

**Discussion by alebcay** [golangci/golangci-lint#4268] [Q&A] [answered]
When using binaries downloaded from release assets (built using `goreleaser`), the version printed in `golangci-lint --version` does not contain the leading `v` prefix. This is because `{{.Version}}` will have its leading `v` stripped off by `goreleaser`.

https://github.com/golangci/golangci-lint/blob/cd0d3447fd7fd9e320a45b3c65d22ecfc814bd8f/.goreleaser.yml#L44

However, the version printed in `golangci-lint --version` when using one of the official container images _does_ contain the leading `v` prefix. This is because the `VERSION` is derived from tag name, without leading `v` stripped off.

https://github.com/golangci/golangci-lint/blob/cd0d3447fd7fd9e320a45b3c65d22ecfc814bd8f/.github/workflows/tag.yml#L92

This is hardly a bug (hence opening this as discussion rather than issue) but I figure it might be nice if these matched regardless of where a user obtains `golangci-lint`.

Happy to open a PR to help fix but would need guidance on which is preferred - keep the `v` prefix or exclude it.

**Comment by ldez** [comment, marked as answer]
Hello,

good catch.

The easier fix is to add the `v` but we can say that the current format (without the `v`) for binaries is the "real expected format" because binaries are the "main" resources.

So I think the "right" solution is to strip the `v` for the Docker image.
